### PR TITLE
Update user profile docs for updatable root attributes

### PR DESCRIPTION
**Pull request by rkhoriander**
Update user profile docs to reflect new updatable attributes on user profile and in signup process.

### CODE REVIEW

**Comment by rkhoriander** [comment]
https://docs-content-staging-pr-7223.herokuapp.com/docs/users

**Comment by rkhoriander** [comment]
@randynasson - Whenever you can provide the screenshot of the new Connection setting in the Dashboard, I can update the guide. Otherwise, I believe this is done. Let me know if you see anything you would like to change.

**Comment by rkhoriander** [comment]
https://docs-content-staging-pr-7223.herokuapp.com/docs/users

File: articles/users/normalized/auth0/normalized-user-profile-schema.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ Fields that are always generated:
...
 * **`user_id`**: the user's unique identifier. This is unique per Connection, but the same for all apps that authenticate via that Connection.
 
-A user's `name`, `nickname`, and `picture` attributes are not directly editable, however you can update the fields in the `user_metadata` to update them for your front-end as desired. [Click here for more details for changing a user's picture](/users/guides/change-user-pictures); the `name`, `nickname` fields can be updated in the same way.
+By default, a user's `name`, `nickname`, and `picture` attributes provided by identity providers other than Auth0 (such as Google, Facebook, Twitter) are not directly editable since they are updated from the identity provider each time a user logs in. If you want to be able to edit these attributes, you must [enable a connection to allow updates to the normalized user profile](/connections/guides/allow-update-normalized-user-profile). When enabled, user attributes will be updated from the identity provider only on user profile creation and will be available to be edited with the Management API's [Update a user endpoint](/api/management/v2#!/Users/patch_users_by_id).
```
Comment (panga) [inline articles/users/normalized/auth0/normalized-user-profile-schema.md]: Please change line 22.

From:
`* **`nickname`**: the user's username.`

To:
`* **`nickname`**: the first part of the user's email or the user's username if using Auth0 store and require username.`

**Review by randynasson — DISMISSED** [review]
Over-all, these changes look good. We don't want to publish them until we are ready to release the feature though. Can we sit on these until we are ready to go live?

File: articles/users/normalized/auth0/normalized-user-profile-schema.md (hunk 1 above)
Comment (rkhoriander) [reply articles/users/normalized/auth0/normalized-user-profile-schema.md]: Done. Thanks!

**Comment by rkhoriander** [comment]
@randynasson: Yes, I have a task in the current sprint, which runs until 3/8. And if you need to wait longer than that, we can move it to the next sprint. I'll continue to check in with you on Slack.

File: articles/users/normalized/auth0/normalized-user-profile-schema.md [hunk 2]
```diff
@@ -18,11 +18,11 @@ The attributes that Auth0 maps to a common schema are listed below.
 Fields that are always generated:
 
 * **`name`**: the user's full name.
-* **`nickname`**: the user's username.
+* **`nickname`**: the first part of the user's email, or the user's username if using the Auth0 data store and require username.
```
Comment (panga) [inline articles/users/normalized/auth0/normalized-user-profile-schema.md]:
```suggestion
* **`nickname`**: the user's username or the local part of the user's email.
```

**Review by panga — CHANGES_REQUESTED** [review]
I need to request changes again.

File: articles/users/normalized/auth0/normalized-user-profile-schema.md (hunk 2 above)
Comment (rkhoriander) [reply articles/users/normalized/auth0/normalized-user-profile-schema.md]: @panga: Are there conditions surrounding when each is used?

Comment (panga) [reply articles/users/normalized/auth0/normalized-user-profile-schema.md]: No, the nickname default value is:
We first try to use username, but if username is not present, we set the local part of the email.
That's why I used the `OR` there.

Comment (rkhoriander) [reply articles/users/normalized/auth0/normalized-user-profile-schema.md]: Updated. Thank you.

**Review by panga — DISMISSED** [review]

**Review by panga — DISMISSED** [review]

**Comment by rkhoriander** [comment]
https://docs-content-staging-pr-7223.herokuapp.com/docs/users/normalized/auth0/update-root-attributes

**Comment by rkhoriander** [comment]
Main pages changed for Bulk Import/Export:

- https://docs-content-staging-pr-7223.herokuapp.com/docs/users/references/user-profile-structure
- https://docs-content-staging-pr-7223.herokuapp.com/docs/users/guides/bulk-user-imports
- https://docs-content-staging-pr-7223.herokuapp.com/docs/users/guides/bulk-user-exports
- https://docs-content-staging-pr-7223.herokuapp.com/docs/extensions/user-import-export
- https://docs-content-staging-pr-7223.herokuapp.com/docs/users/references/bulk-import-database-schema-examples

**Review by tamigoodall — APPROVED** [review]

**Pull request merged**